coord/src/debug_commands.c

```diff
@@ -34,10 +35,17 @@ DEBUG_COMMAND(resumeTopologyUpdater) {
   }
 }
 
+DEBUG_COMMAND(clearTopology) {
+  if (argc != 2) return RedisModule_WrongArity(ctx);
+  RQ_Debug_ClearPendingTopo();
+  return RedisModule_ReplyWithSimpleString(ctx, "OK");
+}
+
 DebugCommandType coordCommands[] = {
   {"SHARD_CONNECTION_STATES", shardConnectionStates},
   {"PAUSE_TOPOLOGY_UPDATER", pauseTopologyUpdater},
   {"RESUME_TOPOLOGY_UPDATER", resumeTopologyUpdater},
+  {"CLEAR_PENDING_TOPOLOGY", clearTopology},
```

Comment:
No changes needed in debug command tests due to this addition?

Reply:
We don't have any test for coordinator debug commands on 2.10 at the moment. only direct usage